### ISSUE TITLE
Problem while trying to install @vue-use-form/yup yup

### ISSUE THREAD

**Issue by viniciusNoleto** [vue-composable-form/vue-composable-form#8]
Hello, I was looking for a form package for Vue and I loved vue-use-form!
I want to use yup validation but I'm having some problems.

Scenario:
Windows 11 Home
Wsl with Ubuntu
Docker container Node
Nuxt 3 Aplication
Npm version: 10.2.4
Node version: v21.6.2

I tryied to install the package many times and always get the same error:

```
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*
```

Complete log:
![Captura de tela 2024-03-22 180826](https://github.com/vue-use-form/vue-use-form/assets/85528669/61e731c3-88b3-4ee9-9c24-35b45e9ad471)

Sorry for bad english [🇧🇷]

**Comment by SnowingFox** [comment]
feel free about your English.

for this project, i don't have much time to mantain it, so this problem may can't be solved.

but it's welcome for you if you want to maintain this project, i will give you permissions about this project.

I'm looking forward to your reply

**Comment by duongductrong** [comment]
Hi @SnowingFox , how can i contribute / maintain this project? I didn't see any document about planning, convention when we joined to contributing this project.

Please let me know about this, i'm using react-hook-form for react and i wanna maintain this project for vue.js

**Comment by SnowingFox** [comment]
> Hi @SnowingFox , how can i contribute / maintain this project? I didn't see any document about planning, convention when we joined to contributing this project.
> 
> Please let me know about this, i'm using react-hook-form for react and i wanna maintain this project for vue.js

yes, you can, you can contact me with x https://x.com/FoxSnowing
